src/app/server/Mdns.cpp

```diff
@@ -122,28 +122,23 @@ CHIP_ERROR AdvertiseOperational()
 /// Set MDNS commissioner advertisement
 CHIP_ERROR AdvertiseCommisioner()
 {
-    return Advertise(false);
+    return Advertise(false, false, false);
```

Comment:
Could we use a bitfield here? figuring out what true/false is seems really hard to follow. We have BitFlags for such things.

Reply:
Or a simple enum like:
```
enum class CommissioningMode
{
  kDisabled,
  kEnabled,
  kEnabledWithAdminCommand
};

Reply:
Was also thinking to consider the previous bool. Looking at `Advertise(false)` even as it stands today, what does it mean? Seems to somehow hint of 'do/do not advertise' but still somewhat vague.

Reply:
I think enum is better than bitfield

Reply:
ok, changed it to an enum since I needed to fix a build breakage